Assets/Controllers/GameManager.cs

```diff
@@ -11,6 +12,7 @@ public class GameManager : MonoBehaviour {
     List<Player> players;
     Player activePlayer;
     PawnKeeper pawnKeeper;
+    int startingNumberOfPlayers = 2;
```

Comment:
You may want to abstract this a bit via setting in a constructor or initializing method. That would keep your declarations clean.